coder_sniffer/Drupal/Sniffs/Commenting/VariableCommentSniff.php

```diff
@@ -111,6 +111,15 @@ public function processMemberVar(File $phpcsFile, $stackPtr)
 
         // The @var tag is the only one we require.
         if ($foundVar === null) {
+            // If there's an inline type argument then you may omit the @var
+            // comment.
+            $propertyName       = $phpcsFile->findNext([T_VARIABLE, T_CLASS, T_INTERFACE], ($commentEnd + 1));
```

Comment:
This should not be needed, because the property name pointer is already in `$stackPtr`, right?

Then I think we can simplify this logic: if the non-whitespace token before $stackPtr is T_STRING then we know that is a type token and we can return.

Reply:
@klausi, my knowledge here are extremely limited, so, no, I don’t get your point, sorry. Actually, it wasn’t me writing this code, I’ve only added test coverage

Reply:
I can check, should not be hard to rewrite this.